include/exadg/grid/grid_utilities.h

```diff
@@ -300,31 +336,13 @@ create_coarse_triangulations(
     if(fine_triangulation.n_global_levels() >= 2)
     {
       // resize the empty coarse_triangulations and coarse_periodic_face_pairs vectors
-      coarse_triangulations = std::vector<std::shared_ptr<dealii::Triangulation<dim> const>>(
-        fine_triangulation.n_global_levels() - 1);
+      std::vector<std::shared_ptr<dealii::Triangulation<dim>>> coarse_triangulations =
+        std::vector<std::shared_ptr<dealii::Triangulation<dim>>>(
+          fine_triangulation.n_global_levels() - 1);
 
       coarse_periodic_face_pairs =
         std::vector<PeriodicFacePairs<dim>>(coarse_triangulations.size());
 
-      // lambda function for creating the coarse triangulations
-      auto const lambda_create_level_triangulation =
```

Comment:
This lambda function has been quite un-intuitive in the present code version in my opinion.